config.yaml

```diff
@@ -0,0 +1 @@
+webhookVersion: 2.0.6+up0.3.6-rc1
```

Comment:
My biggest callout here is to make sure the release checklist gets updated at some point. At least with anything that affects the wider build. For now we own webhook, but once this moves to charts or KDM, the checklist needs to point here.